[AIRFLOW-227] Show environmentally defined configs in the config view

sekikn opened the pull request:
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-227

Testing Done:
- executed run_unit_tests.sh and confirmed all 175 tests passed
- confirmed that the new page:
  - shows no parameter if expose_config is false
  - shows all parameters if expose_config is true, including environment variables such as AIRFLOW__CORE__FOO=bar

Right now the config page (when set to display) just loads the airflow.cfg and lists that,
so any configuration values that are set via an environment variable are not shown.
This patch add a new page that shows all configuration values which are loaded via any means,
only if expose_config is set to true.

msumit commented on airflow/www/templates/airflow/config.html:
```diff
@@ -0,0 +1,67 @@
+    <hr>
+
+    {% if table %}
+        <h5>Running Configuration</h5>
```
imo `h3` would be a better choice here.. also a line break before and after..

sekikn committed:
[AIRFLOW-227] Show running config in config view

Right now the config page (when set to display)
just loads the airflow.cfg and lists that.
So any configuration values that are set via
an environment variable are not shown.

This patch makes all configuration values
which are loaded via any means shown,
only if expose_config is set to true.
